apollo-mockserver/src/commonTest/kotlin/com/apollographql/apollo3/mockserver/test/assertMockResponse.kt

```diff
@@ -2,23 +2,21 @@ package com.apollographql.apollo3.mockserver.test
 
 import com.apollographql.apollo3.api.http.HttpResponse
 import com.apollographql.apollo3.mockserver.MockResponse
+import okio.ByteString
 import kotlin.test.assertEquals
 import kotlin.test.assertTrue
 
 fun assertMockResponse(
     mockResponse: MockResponse,
+    body: ByteString,
```

Comment:
Instead of adding a new parameter, could you do this instead?

```kotlin
  assertEquals(
      mockResponse.body.toList().fold(Buffer()) { buffer, byteString ->
        buffer.write(byteString)
      }.readByteString(),
      httpResponse.body!!.readByteString()
  )
```

This can suspend so you don't want to use it when delays are involved but I think so far it's only used in non-suspending code.

Reply:
Arh it seemed like a good idea, but it's a bit tricky. When using chunked you pass something like `c\r\nFirst chunk\n\r\nc\r\nSecond chunk\r\n0\r\n\r\n` to the `MockResponse` but what you get in the `HttpResponse` is `First chunk\nSecond chunk`. So you can't really compare them as-is.

Reply:
Looks like we need a chunk decoder then ? Should be relatively easy to implement with all the okio/kotlin goodness?